remo/featuredrep/views.py

```diff
@@ -11,11 +11,17 @@
 import forms
 from models import FeaturedRep
 
+from datetime import datetime
+
 
 @cache_control(private=True, max_age=60 * 10)
 def list_featured(request):
     """List all Featured Reps."""
-    featured_list = FeaturedRep.objects.all()
+
+    if not request.user.has_perms('featuredrep.can_edit_featured'):
+        featured_list = FeaturedRep.objects.filter(created_on__lte=datetime.today())
```

Comment:
This seems the origin of errors on test, with this the unit tests are failing maybe because this data is not available.

Reply:
@MichaelKohler do you have any suggestions?

Reply:
As far as I see, the error comes from

```
featured_rep = FeaturedRep.objects.filter(created_on__lte=datetime.today()).latest()
``` 

in ```remo/base/views.py``` and not from here.

I'd guess that the fixtures don't include an entry and therefore .latest() can't return anything and therefore there is no matching query. Adding an entry to the test fixtures should solve this (haven't tried though).

Reply:
https://github.com/mozilla/remo/blob/master/remo/featuredrep/fixtures/demo_featured.json contain an entry and lte is `<=` so have to work.
I am not an expert of fixtures but we cannot have dinamyc fixtures to have the date of today so we can tests also this cases?